doc/api/cli.md

```diff
@@ -511,6 +511,23 @@ added: v2.4.0
 
 Track heap object allocations for heap snapshots.
 
+### `--unhandled-rejections=mode`
+<!-- YAML
+added: REPLACEME
+-->
+
+By default all unhandled rejections trigger a warning plus a deprecation warning
+for the very first unhandled rejection in case no [`unhandledRejection`][] hook
+is used.
+
+Using this flag allows to change what should happen when an unhandled rejection
+occurs. One of three modes can be chosen:
+
+* `strict`: Raise the unhandled rejection as an uncaught exception.
```

Comment:
Can this be `error`?

Reply:
I originally planned on using `error` and `silence` instead of `strict` and `none` but others preferred these and I have no strong opinion on the name in this case.

Reply:
nit: may want to add a quick statement that the `mode` is case-sensitive.